lib/smart_proxy_ansible/roles_reader.rb

```diff
@@ -5,7 +5,7 @@ module Ansible
     # Implements the logic needed to read the roles and associated information
     class RolesReader
       class << self
-        DEFAULT_ROLES_PATH = '/etc/ansible/roles:/usr/share/ansible/roles'.freeze
+        DEFAULT_ROLES_PATH = '/etc/ansible/roles:/usr/share/ansible/roles:/var/lib/foreman-proxy/ansible/roles'.freeze
```

Comment:
I'd think this should read first from the mutable directory so VCS cloned repos can override system wide repos. See the overall review for more on settings.